src/test/java/jenkins/plugins/office365connector/workflow/WebhookJobPropertyTest.java

```diff
@@ -1,16 +1,19 @@
 package jenkins.plugins.office365connector.workflow;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.*;
```

Comment:
Prefer not to use asterix

Reply:
fixed

Reply:
not really, now this import is redundant